FF1Blazorizer/Pages/Randomize.razor

```diff
@@ -28,7 +28,7 @@
 		<PartyTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
 		<ConveniencesTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
 		<BugFixesTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" />
-		<ExperimentalTab Flags="@Flags" UpdateToolTipID="@UpdateToolTipID" IsOpen="false" />
```

Comment:
This was deliberate of course. Now that there is like 4 things we don't expect people to use I wanted to hide it by default.

Reply:
Oh yeah, but this way when you tried to click on a flag it in Experimental it kept closing it, it's still hidden by default with this change.

Reply:
Oh that's crazy. Thanks!